### PR TITLE
Init korean version

### PR DESCRIPTION
**Pull request by jeyong**
I add ./kr directory for Korean version.
I skipped a few pages but I'm working in progress.

### CODE REVIEW

**Comment by hamishwillee** [comment]
Fantastic - thank you. Let me know when you are ready to merge!

File: kr/SUMMARY.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  * [유지보수](test_and_ci/maintenance.md)
+
+
+  
+## Dronecode 바로가기
+
+* [PX4 사용자 가이드](https://docs.px4.io/en/)
+* [QGroundControl 사용자 가이드](https://docs.qgroundcontrol.com/en/)
+* [QGroundControl 개발자 가이드](https://dev.qgroundcontrol.com/en/)
+* [MAVLink 개발자 가이드](https://mavlink.io/en/)
```
Comment (hamishwillee) [inline kr/SUMMARY.md:151]: This link can go here: https://mavlink.io/kr/ - after all, you did all that work to translate it :-)

File: kr/SUMMARY.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  * [지속통합](test_and_ci/continous_integration.md)
+    * [Jenkins 지속통합](test_and_ci/jenkins_ci.md)
+  * [유지보수](test_and_ci/maintenance.md)
+
+
+  
+## Dronecode 바로가기
+
+* [PX4 사용자 가이드](https://docs.px4.io/en/)
+* [QGroundControl 사용자 가이드](https://docs.qgroundcontrol.com/en/)
```
Comment (hamishwillee) [inline kr/SUMMARY.md:149]: This should go to your translation (as it does exist): https://docs.qgroundcontrol.com/kr/

**Comment by hamishwillee** [comment]
I just made quite a few fixes in #193. Can you make the same modifications I made to the English files to the Korean tree (these were resulting in lots of reported errors).

**Comment by hamishwillee** [comment]
I will confirm with the rest of the team if there are any gating factors to merging.

**Comment by jeyong** [comment]
I fixed #193.

**Comment by hamishwillee** [comment]
@LorenzMeier @bkueng Actually I did merge - it is good to go. @jeyong thank you so much for this contribution. It is live now :-)

**Comment by jeyong** [comment]
Thank you. :)

**Pull request merged**